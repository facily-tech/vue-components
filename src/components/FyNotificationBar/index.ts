export { default as FyNoticationBar } from './FyNotificationBar.vue';
