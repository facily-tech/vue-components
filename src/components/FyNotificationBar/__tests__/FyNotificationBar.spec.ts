import Vue from 'vue';
import Vuetify from 'vuetify';
import FyNotificationBar from '../FyNotificationBar.vue';
import { mount, MountOptions, Wrapper } from '@vue/test-utils';

Vue.use(Vuetify);

describe('FyNotificationBar.ts', () => {
  // eslint-disable-line max-statements
  type Instance = InstanceType<typeof FyNotificationBar>;
  let mountFunction: (options?: MountOptions<Instance>) => Wrapper<Instance>;
  beforeEach(() => {
    mountFunction = (options?: MountOptions<Instance>) => {
      return mount(FyNotificationBar, {
        vuetify: new Vuetify(),
        ...options,
      });
    };
  });

  it('should render component and match snapshot', () => {
    const wrapper = mountFunction();

    expect(wrapper.html()).toMatchSnapshot();
  });
});
