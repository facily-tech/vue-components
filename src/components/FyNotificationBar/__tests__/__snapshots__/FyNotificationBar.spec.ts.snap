// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FyNotificationBar.ts should render component and match snapshot 1`] = `
<div class="fy-notication-bar">
  <div class="v-menu"><button type="button" class="v-btn v-btn--icon v-btn--round theme--light elevation-0 v-size--default" primary="" icon=""><span class="v-btn__content"><i aria-hidden="true" class="v-icon notranslate mdi mdi-bell theme--light"></i></span></button>
    <!---->
  </div>
</div>
`;
